README.rst

```diff
@@ -19,11 +19,15 @@ Installation
 
```

Comment:
Any reason you are using .rst instead of .md?

Reply:
I think this is because the docs build from `cheshire` assumes everything is RST.

Reply:
Yeah I'm just following the patterns from `cheshire` here

Reply:
I think this package is generically useful enough that it would be good to write some basic docs explaining what it's doing and publish on RTD, so I think this is the right move.